client/my-sites/plans/main.jsx

```diff
@@ -31,7 +31,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
I'm actually keeping this default (and removed the one in `PlansFeaturesMain`) because this component (`my-sites/plans/main`) is used only for the `/plans` page (and not in wpcom or Jetpack NUX). If, for some reason, we don't have all the needed props to compute `displayJetpackPlans` in reducer, we risk running into errors. That's why I think we should default here to `false` (so show wpocm plans on `/plans` page by default).

What do you think @johnHackworth?